build/lib/release.sh

```diff
@@ -396,6 +396,7 @@ EOF
             kube::log::status "Tagging docker image ${docker_image_tag} as ${release_docker_image_tag}"
             "${DOCKER[@]}" rmi "${release_docker_image_tag}" 2>/dev/null || true
             "${DOCKER[@]}" tag "${docker_image_tag}" "${release_docker_image_tag}" 2>/dev/null
+            "${DOCKER[@]}" save "${docker_image_tag}" "${release_docker_image_tag}" -o "${binary_dir}/${binary_name}.tar"
```

Comment:
So this is doing the same thing as L358, but with a different tag?

Reply:
this is doing the same as L385, but with multiple tags in the same archive

Reply:
Could we change the logic to do this once, instead of twice? This works, but is inefficient because it possibly does a docker save twice

Reply:
I can probably move the line with the top if for KUBE_DOCKER_IMAGE_TAG, before we do the first save and save a local var with the additional args for save.
Is this what you have in mind?

Reply:
Something like that, yeah. And omit the second tag if it's not any different than the docker image tag.

Reply:
done